tools/batch/batch_states/compile_notebook.py

```diff
@@ -0,0 +1,100 @@
+import os
+import argparse
+import sys
+import re
+
+HOME_PATH = os.path.expandvars('$HOME') + "/work/gluon-nlp"
+
+class DeployWebsiteDriver(object):
+    def __init__(self, args):
+        self.md_file_list = []
+        if args.pr_number != '0':
+            self.pr_number = 'PR#' + args.pr_number
+        else: 
+            self.pr_number = 'push'
```

Comment:
is there a build number similar to jenkins?

Reply:
There is `github.run_number` for each run of a particular workflow.

Reply:
let's make sure it's in the path.